database/users_quiz_progress_table.py

```diff
@@ -25,3 +25,17 @@ async def insert_user_answer(user_id, quiz_id, completed, score):
                          parameters=(user_id, quiz_id, completed, score)
                          )
         await db.commit()
+
+
+async def fetch_user_stat_by_topic(user_id):
```

Comment:
I think the name of the function "fetch user achievements" better reflects its task.